### ISSUE TITLE
📎 Support prettier-style jsx quotes

### ISSUE THREAD

**Issue by nstepien** [rome/tools#4486]
### Description

Since Rome 12.1.0, Rome started enforcing the same quote style on JSX prop strings.
This breaks parity with prettier: with `"singleQuote": true` prettier would still format JSX strings with double-quotes.

[Playground example](https://docs.rome.tools/playground/?quoteStyle=single&code=PABkAGkAdgAgAHQAaQB0AGwAZQA9ACIAYQAiACAALwA%2BADsA)

Straying away from prettier as the "standard" makes it a hard pill to swallow as projects would have to reformat the majority of their files.

Possible approaches:
- revert back to always formatting JSX strings with double quotes
- add a new option, something like `"jsxQuoteStyle": "double" / "single"`, `"double"` by default.
  I think it's a fairly safe option to add as it wouldn't affect the formatting of other parts of the code, it'd only change the quote characters and quote escaping in the strings.

Previous discussion: https://github.com/rome/tools/issues/4469

**Comment by johnschult** [comment]
I agree, this started reformatting all our JSX, which was not optimal.

**Comment by Vivalldi** [comment]
As an alternative option naming, `markupQuoteStyle`. Which pairs well with XML (JSX is JavaScript XML), HTML & YAML

**Comment by vascomontyero** [comment]
what is the status of this? wanted to update to version 12, but this is blocking us

**Comment by denbezrukov** [comment]
I could take it to implement🤗

**Comment by afc163** [comment]
```
  "javascript": {
    "formatter": {
      "quoteStyle": "single",
      "jsxQuoteStyle": "double"
    }
  }
```

This don't work as expected.

- Expected: 

```jsx
const App = () => {
  const stringVar = '123'; // quoteStyle works
  return <div id="xxx" />; // jsxQuoteStyle works
}
```


- Current behavior: 

```jsx
const App = () => {
  const stringVar = "123"; // jsxQuoteStyle works here now, quoteStyle would be useless
  return <div id="xxx" />; // jsxQuoteStyle works
}
```

**Comment by ematipico** [comment]
This feature hasn't been released yet, but it works as expected: https://docs.rome.tools/playground/?quoteStyle=single&code=YwBvAG4AcwB0ACAAQQBwAHAAIAA9ACAAKAApACAAPQA%2BACAAewAKACAAIABjAG8AbgBzAHQAIABzAHQAcgBpAG4AZwBWAGEAcgAgAD0AIAAnADEAMgAzACcAOwAgAC8ALwAgAHEAdQBvAHQAZQBTAHQAeQBsAGUAIABzAGkAbgBnAGwAZQAKACAAIAByAGUAdAB1AHIAbgAgADwAZABpAHYAIABpAGQAPQAiAHgAeAB4ACIAIAAvAD4AOwAgAC8ALwAgAGoAcwB4AFEAdQBvAHQAZQBTAHQAeQBsAGUAIABkAG8AdQBiAGwAZQAKAH0AKgA%3D

I would need to use a nightly release to avail of the feature.

**Comment by nexckycort** [comment]
This functionality is not available in the version 12.1.3?
